CONTRIBUTING.md

```diff
@@ -10,6 +10,7 @@ Once your user is created and added to the org this guide details the steps to p
 
 * rebase and get  your branch ahead of master
 * then from your working branch
+* run `npm audit --audit-level=high` to ensure there are no vulnerabilities with level high or critical.
```

Comment:
@bradjteamsnap is this sufficient or expand more?

Reply:
Works for me :)